README.md

```diff
@@ -1,2 +1,219 @@
+- Find the country,
+- See the current air pollution gases in this country.
+
+## 🛠 Built With 
+<a name="built-with"> React </a>
+<a name="built-with"> Redux </a>
+<a name="built-with"> React Bootstrap</a>
+<a name="built-with"> Hooks</a>
+
+### Tech Stack 
+<a name="tech-stack">React Redux</a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://react.org/">React</a></li>
+    <li><a href="https://css.org/">Redux</a></li>
+    <li><a href="https://css.org/">React Bootstrap</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Render</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+   <li><a href="https://restcountries.com/">Restcountries API</a></li>
+   <li><a href="https://openweathermap.org/api/air-pollution">OpenWeatherMap API</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Show-Countries-per-continenet]**
+- **[Show-pollution-Rate]**
+- **[show-gas-combination]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Screenshots
+
+## Countries_View
+![Countries_View](./src/assets/continents-screenshoot.png)
+
+## Pollution_View
+![Pollution_View](./src/assets/Pollution-screenshoot.png)
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://airpollutionapp.onrender.com/)
+Coming soon!
+
+## Demonstration Video
+- [Demo-Video](https://www.loom.com/share/b53c268cca8b41adb0c7a3f4650cf8eb)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+- This project was bootstrapped with Create React App.
+- Clone this repo to your local machine 
+- cd into the folder and run npm install to install all the dependencies.
+- Run npm start to start the app.
+- Open http://localhost:3000 to view it in the browser.
+- The page will reload if you make edits.
+- You will also see any lint errors in the console.
+- Run npm run build to build the app for production to the build folder.
+- It correctly bundles React in production mode and optimizes the build for the best performance.
+- The build is minified and the filenames include the hashes.
+- Your app is ready to be deployed!
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Have visual studio Code in your PC.
+- Have a verified GitHub account.
+- A code editor
+- Nodejs
+- NPM
+- Have a stable browser version.
+
+### Setup
+
+- Create a local directory where you can clone the project.
+- Clone this repository to your desired folder using terminal: 
+git clone https://github.com/ReemMohamedAbdelfatah/air-pollution-app.git
+- Install the dependencies
+- Open the project on a browser
+
+### Install
+
+Install this project with:
+
+- Just run npm i to install all dependencies
+
+### Usage
+
+To run the project, execute the following command:
+
+- npm run build
+- npm run start
+
+### Run tests
+
+To run tests, run the following command:
```

Comment:
- [ ] [ OPTIONAL]
Kindly include appropriate instructions under the Getting Started section of your ReadMe, especially on how to _run test_ cases for this project.

Reply:
Done 👌